src/main/java/frc/robot/Constants.java

```diff
@@ -59,13 +62,27 @@ public class Constants {
                     MAX_ANGULAR_VELOCITY.in(Units.RotationsPerSecond),
                     MAX_ANGULAR_ACCELERATION.in(Units.RotationsPerSecond.per(Units.Second)));
 
-    public static final Pose2d[] OPTIMAL_POINTS_SHOOT = {
-        new Pose2d(1.97, 7.16, Rotation2d.fromDegrees(-161.57)),
-        new Pose2d(2.54, 3.06, Rotation2d.fromDegrees(-157.15)),
-        new Pose2d(4.16, 5.03, Rotation2d.fromDegrees(179.94))
-    };
-    public static final Pose2d[] OPTIMAL_POINTS_TRAP = {};
-    public static final Pose2d AMP_POSE = new Pose2d(0.0, 0.0, new Rotation2d(0, 0));
+    public static final List<Pose2d> OPTIMAL_POINTS_SHOOT_BLUE =
+            List.of(
+                    new Pose2d(1.97, 7.16, Rotation2d.fromDegrees(-161.57)),
+                    new Pose2d(2.54, 3.06, Rotation2d.fromDegrees(-157.15)),
+                    new Pose2d(4.16, 5.03, Rotation2d.fromDegrees(179.94)));
+    public static final List<Pose2d> OPTIMAL_POINTS_SHOOT_RED =
+            OPTIMAL_POINTS_SHOOT_BLUE.stream()
+                    .map(GeometryUtil::flipFieldPose)
+                    .collect(Collectors.toList());
+
+    public static final List<Pose2d> OPTIMAL_POINTS_TRAP_BLUE =
+            List.of(new Pose2d(0, 0, Rotation2d.fromDegrees(0)));
```

Comment:
Is there only one point? If yes, this doesn't have to be a list.

Reply:
The points will be added later